go.mod

```diff
@@ -187,6 +187,7 @@ require (
 )
 
 replace (
+	github.com/celestiaorg/go-header => github.com/gupadhyaya/go-header v0.0.0-20230623130312-b9d75475a548
```

Comment:
?? was this for when a PR was opened?

Reply:
nope. last night fix commit added that. it will be updated once https://github.com/celestiaorg/go-header/pull/57 is merged.

Reply:
Yes, this change was part of the fix from @gupadhyaya and for which this PR: https://github.com/celestiaorg/go-header/pull/57 was opened. Ideally, we have a new go-header version once https://github.com/celestiaorg/go-header/pull/57 is merged and we can use that new go-header version for this rollkit PR.

Reply:
https://github.com/celestiaorg/go-header/pull/57 is merged we can do a release in go-header